File: src/main/java/com/pCarpet/model/Experience.java
package com.pCarpet.model;
import java.util.List;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToMany;
import javax.validation.constraints.NotNull;

import org.hibernate.annotations.OnDelete;
import org.hibernate.annotations.OnDeleteAction;
import org.springframework.lang.Nullable;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity


public class Experience {
	@Id
	@Column(name = "idExperience")
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer idExperience;
	
	@ManyToOne
	@JoinColumn(name="idUser")
	private User user;
	
	@Column(name = "commento")
	private String commento;
	
	@Column(name = "positivo")
	private String positivo;
	
	@Column(name = "negativo")
	private String negativo;
	
	@Column(name = "score")
	@NotNull
	private Integer score;
	
	@OneToMany(mappedBy="experience")
	@OnDelete(action=OnDeleteAction.CASCADE)
	private List<Feedback> feedback;
	
	
	
	



}

File: src/app/models/Experience.ts
import  {Feedback} from './Feedback';
import  {User} from './User';
import  {Imagen} from './Imagen';

export class Experience {
    idExperience: number;
    user: User;
    idPrincipi: number;
    commento: string;
    positivo: string;
    negativo: string;
    score: number;
    feedback: Array<number>;
    
    

    // tslint:disable-next-line:max-line-length
    constructor(user: User, idPrincipi: number, commento: string, positivo: string, negativo: string,  score: number, feedback: Array<number>) {
        this.idPrincipi =  idPrincipi;
        this.user = user;
        this.commento = commento;
        this.positivo =positivo;
        this.negativo = negativo;
        this.score = score;
        this.feedback =  feedback;
        
    }

}